# Configuration to be used for the different tasks.
# Possible robot_name to set, depending on the task :
# - ur5_robot
# - ur10e_robot
#
# Possible pose to set :
# - ee_pose_work_offset : Position offset as a safe position to engage / disengage
#   the task in the environment in the robot base_link frame
# - ee_pose_offset : Position offset to perform the task in the environment in the
#   robot base_link frame
# - ee_pose_scan : Pose to scan the environment, in the robot base_link frame
#
# Louis Munier : 2024-12-04

welding:
  robot_name: "ur10e_robot"
  ros_version: "noetic"
  home_configuration: [1.57, -1.57, 0, -1.57, 0, 0] # [rad] j1, j2, j3, j4, j5, j6
  working_angle: 0.3491 # [rad]
  working_speed: 0.012 # [m/s]
  ee_pos_work_offset: [0, 0, -0.1] # [m] xyz
  ee_pos_offset: [0, 0, -0.02] # [m] xyz
  ee_pose_scan: [-0.702306, -0.0164152, 0.0117277, 0.711589, -0.00866204, 0.510377, 0.632986] # [rad] xyzw, [m] xyz
  # ee_pose_scan: [-0.702306, -0.0164152, 0.0117277, 0.711589, -0.50866204, 0.210377, 0.632986] # [rad] xyzw, [m] xyz
cleaning:
  robot_name: "ur5_robot"
  ros_version: "noetic"
  home_configuration: [0, -1.57, 0, -1.57, 0, 0] # [rad] j1, j2, j3, j4, j5, j6
  working_angle: 0.3491 # [rad]
  working_speed: 0.1 # [m/s]
  ee_pos_work_offset: [0, 0, -0.3] # [m] xyz
  ee_pos_offset: [0, 0, -0.02] # [m] xyz
  ee_pose_scan: [0, 0, 0, 1, 0.5, 0.5, 0.5] # [rad] xyzw, [m] xyz
